pages.de/android/dalvikvm.md

```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> Android Java virtuelle Maschine.
+> Mehr Informationen: <https://source.android.com/devices/tech/dalvik>.
+
+- Starte ein Java Programm:
```

Comment:
In German, compound words are generally not separated by spaces. (pejoratively, this mistake is also referred to as the "Idioten Leerzeichen")
```suggestion
- Starte ein Java-Programm:
```